src/utils/DateCalculate.js

```diff
@@ -0,0 +1,67 @@
+export function compareDateDifferences(rawDataTime) {
+  const currentTime = new Date();
+  const time = new Date(rawDataTime);
+
+  const timeDifference = currentTime - time; // 시간 차
+
+  const result = answerDateDifferences(destructureTime(timeDifference));
```

Comment:
아래와 같은 느낌으로 함수명에서 함수의 행위가 더 명확하게 드러나야 할 것 같은데요~
answer라는 네이밍이 어색하게 느껴지는게 큰 것 같아요.
```suggestion
  const result = getDateDifferencesLabel(destructureTime(timeDifference));
```

Reply:
destructureTime이라는 이름은 함수의 동작이 드러나지 않는 것 같아요.